tensorflow_datasets/scripts/generate_visualization.py

```diff
@@ -0,0 +1,32 @@
+"""
+Script for generating dataset example figures
+
+Uses show_examples() to generate example figures of a dataset
+and stores it temporary in examples/
+
+Typical usage example:
+
+  ds_name = 'beans'
+  generate_visualization(ds_name)
+
+"""
+
+import tensorflow_datasets as tfds
+import matplotlib.pyplot
+
+FIG_DIR = ('examples/')
```

Comment:
Those images are intended to be displayed in the doc, so should go under `docs/catalog/images/` or something like that.